terracumber_config/tf_files/MLM-5.1-build-validation-NUE.tf

```diff
@@ -0,0 +1,1213 @@
+// Mandatory variables for terracumber
```

Comment:
Same about NUE and PRG.

Reply:
From Eric:
>No hard feelings.
I do not plan to start the infra DNS part any time soon, because it's a lot of work, because it needs cooperation from external infra, and because it has a potential for breaking everything, but for pipelines files, no strong objection.

Leaving it as is for now to be consistend with the rest.